### DISCUSSION TITLE
Ores in the Nether

### DISCUSSION THREAD

**Discussion by cha0s5150** [lukebemish/excavated_variants#153] [Q&A] [answered]
I'm trying to make life a little easier for Origins that start in the Nether on my server. This seems like the mod for me to use to get that done.
I've already modified some of the configs to test if this will work and now I can see blocks for Blackstone Iron Ore and Basalt Iron Ore.
The problem I am having is getting these blocks to now spawn naturally in the Nether. I feel like I am missing something, but I am unsure where it might be.
Any help would be appreciated. Thanks.

**Comment by lukebemish** [comment, marked as answer]
Excavated Variants doesn't cause more ores to spawn; it only replaces ores that spawn with versions that match the stone they are in. You'll need either a datapack or a mod (KubeJS could do it) to add ores to the nether, and then let Excavated Variants handle the stone variants

**Reply by cha0s5150** [reply]
Good to know. Thank for the information. I'll see what I can do to get that working.
